src/Microsoft.ML.Data/DataLoadSave/CompositeDataReader.cs

```diff
@@ -90,7 +90,7 @@ public static class CompositeDataReader
         /// <summary>
         /// Save the contents to a stream, as a "model file".
         /// </summary>
-        public static void SaveTo<TSource>(this IDataReader<TSource> reader, IHostEnvironment env, Stream outputStream)
+        public static void SaveTo<TSource>(this IDataLoader<TSource> reader, IHostEnvironment env, Stream outputStream)
```

Comment:
```suggestion
        public static void SaveTo<TSource>(this IDataLoader<TSource> loader, IHostEnvironment env, Stream outputStream)
```

Reply:
thank you!

---
In reply to: [260381588](https://github.com/dotnet/machinelearning/pull/2731#discussion_r260381588) [](ancestors = 260381588)